api-reference/_hidden/dxPolarChartCommonAnnotationConfig/series.md

```diff
@@ -0,0 +1,26 @@
+Use this option when the annotation is positioned relative to an [argument](/api-reference/_hidden/dxChartCommonAnnotationConfig/argument.md '{basewidgetpath}/Configuration/annotations/#argument').
```

Comment:
"is positioned relative to" Explain

Reply:
![image](https://user-images.githubusercontent.com/33040934/74844006-dc28e700-533d-11ea-8fbc-cc432309744b.png)
![image](https://user-images.githubusercontent.com/33040934/74844026-e0ed9b00-533d-11ea-887a-d00dbd58ee78.png)

Reply:
Ok